### PR TITLE
Update customer-experience-toolkit.md

### PR DESCRIPTION
**Pull request by ToniBonittoGSA**
This PR implements the following **changes:**

Update links in https://digital.gov/resources/customer-experience-toolkit/ 

**URL / Link to page**

Preview: 
https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.app.cloud.gov/preview/gsa/digitalgov.gov/cx-updates/resources/customer-experience-toolkit/

### CODE REVIEW

File: content/resources/customer-experience-toolkit.md [hunk 1]
```diff
@@ -102,22 +113,23 @@ Publish customer service standards and train all employees so they understand th
 **Train your employees**
```
Comment (RachelF) [inline content/resources/customer-experience-toolkit.md:113]: The sub-headings in the "culture" section don't match the formatting of the other sections, please check.

File: content/resources/customer-experience-toolkit.md [hunk 2]
```diff
@@ -281,14 +317,16 @@ Develop a Digital Business Strategy that embraces an improved customer experienc
...
 - Coordinate across all channels (web, social, mobile, etc.)
     - [U.S. Digital Registry](https://digital.gov/services/u-s-digital-registry/)
+    - [Join the Social Media Community of Practice](https://digital.gov/communities/social-media/)
 
 ### Communicate clearly
 
 - [Plain Language and Writing for the Web](https://www.youtube.com/watch?v=j9FCji6-UJE) &#8211; video
+- [Join the Federal Communicators Network](https://digital.gov/communities/federal-communicators-network/)
 
 ### Adopt shared services
```
Comment (RachelF) [inline content/resources/customer-experience-toolkit.md:327]: Suggest adding a second bullet to this section: 
Customer Experience Services Evaluation and Buying Guide 
(https://www.performance.gov/cx/assets/files/CX-Evaluation-and-Buying-Guide.pdf) 
(PDF, 823 KB, 2020) - guidance on purchasing Human-Centered Design (HCD) services

**Review by RachelF — DISMISSED** [review]
looks good, thank you!!

**Comment by ToniBonittoGSA** [comment]
> looks good, thank you!!

Got 'em, thanks!

**Pull request merged**